src/containers/app/index.js

```diff
@@ -111,14 +111,12 @@ class App extends React.Component {
     }
 
     render() {
-        const mainPath = (this.props.location.pathname.split('/').length > 0)
-            ? this.props.location.pathname.split('/')[1]
-            : '';
+        const mainPath =
+            this.props.location.pathname.split("/").length > 0 ? this.props.location.pathname.split("/")[1] : "";
 
         const showConnection =
-            ["account", "exchange", "loan", "reserves", "lock", "tryit", "loan"].indexOf(
-                mainPath
-            ) > -1;
+            ["account", "exchange", "loan", "reserves", "lock", "tryit", "loan"].indexOf(mainPath) > -1;
+        const hideMsgPanel = ["", "concept", "roadmap", "manifesto", "disclaimer", "contact"].indexOf(mainPath) > -1;
```

Comment:
would it make sense only showing the MsgPanel on connected pages? in that case you don't need to introduce a new variable just wrap the panels around with the `if (showConnection)` condition

Reply:
@szerintedmi 
Do we want to show msgPanel on the under-the-hood page?